### DISCUSSION TITLE
[Request] Aqara Smart Wall Switch H1 - LED configuration?

### DISCUSSION THREAD

**Discussion by eScriptor** [Koenkk/zigbee2mqtt#9864] [Devices]
Hello

I acquired some Xiaomi Aqara Smart Wall Switch H1, I'm satisfied with the Zigbee2MQTT integration but unfortunately there is no settings available to manage the LED of the switch.
By default, the blue LED turns on when the relay is closed, but I would have preferred the opposite.

I think the Xiaomi App is capable to change this settings, so there has to be a Zigbee command to manage this.

**Comment by octa22** [comment]
Hi, I have created a PR to the zigbee-herdsman-converters (https://github.com/Koenkk/zigbee-herdsman-converters/pull/3779) which adds the support for the led light flipping for Aqara H1 switches.
I hope it gets merged soon.

**Comment by eScriptor** [comment]
Thank you very much !
